assets/styles/gambit.css

```diff
@@ -0,0 +1,36 @@
+body {
```

Comment:
You can probably use the [Chrome](http://forge.dosomething.org/#chrome) pattern from Forge for this!